src/Components/TrainingPrograms/TrainingPrograms.js

```diff
@@ -114,19 +116,41 @@ function TrainingPrograms() {
     temp[index][element] = event.target.value;
     setAllPrograms(temp);
     console.log(allPrograms);
+    console.log(date);
   };
 
-  const editDate = (newValue, label, index) => {
-    console.log(newValue);
-    const temp = [...allPrograms];
-    temp[index][label] = newValue;
-    setAllPrograms(temp);
-    console.log(allPrograms);
-  };
+  // useEffect(() => {
+  //   const temp = date;
+  //   setAllPrograms(
+  //     {
+  //       ...allPrograms,
+  //       referral: temp,
+  //     },
+  //   );
+  // }, [date]);
+  
+
+  // const editDate = (newValue, label, index) => {
+  //   console.log(newValue);
+  //   const temp = [...allPrograms];
+  //   temp[index][label] = newValue;
+  //   setAllPrograms(temp);
+  //   console.log(allPrograms);
+  // };
 
   return (
     <div className="content">
       <div className="tp-title">Training Programs</div>
+      <div className="alert-modal">
+        <p>
+          You've marked this milestone as complete for Jobseeker-Name. Click
+          {' '}
+          <a href="#">here</a>
```

Comment:
sorry solia for getting sloppy
but since its kinda just a temporary (front end only) pr, i'll merge for now and this issue will be replaced with a button
the comments and stuff i haven't been keeping track of so we'll have another bugbathpatch when i'm free